okio/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -135,7 +135,7 @@ abstract class Filesystem {
   }
 
   /**
-   * Returns the children of the directory identified by [dir].
+   * Returns the children of the directory identified by [dir]. The returned list is sorted.
```

Comment:
Is it worth specifying that it's "sorted using natural ordering"?

Reply:
Fixed.